### DISCUSSION TITLE
App Share

### DISCUSSION THREAD

**Discussion by AliAsgerSaify** [iamrasel/lunar-launcher#83] [Ideas]
I have Oppo phone, so when we long press on an app in the menu there is an option to share that app. So in order to do that I have to switch launcher, then share , then change launcher again.
 So Can we please have this feature in lunar launcher as well.

By the way nice minimalistic launcher.

Thanks in advance

**Comment by iamrasel** [comment]
Could you please tell me more about this `share` option? How does it share? Does it share the app file, or the link to app store?

**Reply by AliAsgerSaify** [reply]
It share the apk file for that app, shown in the attached ss.!

[Screenshot_2023-08-11-19-06-05-09_6012fa4d4ddec268fc5c7112cbb265e7.jpg](https://github.com/iamrasel/lunar-launcher/assets/120162464/156ff850-7b4c-4fad-b4d5-d3989bc50920)

**Reply by iamrasel** [reply]
Okay, I'll try to add this soon.

**Reply by AliAsgerSaify** [reply]
Thank you for you attention,

Thanks in advance

**Comment by iamrasel** [comment]
Almost done, wait for the next update.

**Reply by AliAsgerSaify** [reply]
When will it be available on Fdroid to download?

**Reply by iamrasel** [reply]
@AliAsgerSaify Haven't released the related update yet. So, it should be pretty much available in about 1-2 weeks.

**Reply by AliAsgerSaify** [reply]
Hello,
Thanks for the update but this is not what I asked for, you already clarified this part but still. I want to share the app's apk file not the link to download. I am asking for this because I rarely use app from playstore, I use it from fdroid and others dont have fdroid, so its easier to share the apk file directly.

Anyways thanks for the consideration. And I hope you have considered this much, then you will consider improvising this feature too. ☺️

**Reply by iamrasel** [reply]
Which version are you talking about? The thing you have asked for has been added in version 2.8. F-droid's release cycle is too slow that the version hasn't reached there yet; wait few more days or update through GitHub release.

**Reply by AliAsgerSaify** [reply]
**version: 2.7.2**

**Reply by iamrasel** [reply]
Then wait for the next update from F-droid. Though I had pushed the updated version almost two weeks ago.

**Reply by AliAsgerSaify** [reply]
I have got the latest update. Thanks for the consideration.
And regarding previous reply, I am a bit curious about what is the procedure of you publishing the application until it comes live on F-droid.

**Reply by iamrasel** [reply]
The proper guide on submitting an app to F-droid can be found [here](https://f-droid.org/contribute/). After the app gets added to F-droid's list, everytime an update gets released on Github with a tag that matches the version code, F-droid includes the app in it's building cycle. The time to finish the building and publishing cycle for an individual app can take several days or weeks.
